core/src/test/java/org/apache/iceberg/util/TestStructLikeMap.java

```diff
@@ -43,28 +43,29 @@ public void testSingleRecord() {
     Record record1 = gRecord.copy(ImmutableMap.of("id", 1, "data", "aaa"));
 
     Map<StructLike, String> map = StructLikeMap.create(STRUCT_TYPE);
-    Assert.assertEquals(0, map.size());
+    assertThat(map).isEmpty();
 
     map.put(record1, "1-aaa");
-    Assert.assertEquals(1, map.size());
-    Assert.assertFalse(map.isEmpty());
-    Assert.assertTrue(map.containsKey(record1));
-    Assert.assertTrue(map.containsValue("1-aaa"));
-    Assert.assertEquals("1-aaa", map.get(record1));
+    assertThat(map.size()).isOne();
+    assertThat(map)
+        .isNotEmpty()
+        .containsKey(record1)
+        .containsValue("1-aaa")
+        .containsEntry(record1, "1-aaa");
 
     Set<StructLike> keySet = map.keySet();
-    Assert.assertEquals(1, keySet.size());
-    Assert.assertTrue(keySet.contains(record1));
+    assertThat(keySet.size()).isOne();
+    assertThat(keySet).contains(record1);
```

Comment:
can be combined to `assertThat(keySet).hasSize(1).contains(record1)`.
Please also do the same for all the other places